packages/training/book/src/scoring/rubrics.md

```
@@ -0,0 +1,272 @@
+{
+  "rubrics": {
+    "my-archetype": "## My Archetype Evaluation\n\n### What Makes an Excellent...",
+  },
+  "priorityMetrics": {
+    "my-archetype": ["trading.totalPnL", "custom.metric"]
+  },
+  "availableArchetypes": [..., "my-archetype"]
+}
+```
+
+2. Export and validate:
```

markdownlint-cli2 (0.18.1)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)